docs/source/developers/documentation.rst

```diff
@@ -94,7 +94,7 @@ Docker container.
 
 .. code-block:: shell
 
-  archery docker run ubuntu-docs
+  archery docker run -v "$(PWD)/docs:/build/docs" ubuntu-docs
 
 The final output is located under ``docs/_build/html``.
```

Comment:
@amol- This may not be valid due to the mount. Can you confirm?

Reply:
Uhm, not sure I get what you mean. If you are talking about the final output location, once the `-v` option is provided it's correct as we are building the local `docs` directory as `/build/docs` where the actual output is written.

Reply:
I've just fixed the build script to generate the docs within the container directly to `/build/docs`.